lib/util/env.js

```diff
@@ -0,0 +1,23 @@
+function getUntrustedEnv(...customEnvVars) {
+  const envVars = [
+    'HTTP_PROXY',
+    'HTTPS_PROXY',
+    'NO_PROXY',
+    'HOME',
+    'PATH',
+    ...customEnvVars,
```

Comment:
Does this syntax definitely work? I thought passing an array of variables (as the one and only parameter) is maybe clearer.

Reply:
I've suggested the interface change above. Also for each variable we want to check first that it's defined, e.g. `if (typeof process.env[var] !== 'undefined')` similar to https://github.com/renovatebot/renovate/pull/4015/files#diff-aa656697b9687f2941a8a6da52236c18R13

Reply:
Yes most probably this function does the same thing, It first checks for environment variables definitions in Capital letters, if undefined it checks for lower case. If it's not defined it not set.This is the intent right?
And yes syntax works but it's not readable. I ll refactor it.